demos/rag_agentic/src/agent_with_mcp_ocp_slack.py

```diff
@@ -0,0 +1,110 @@
+# Code bellow written following examples here: https://llama-stack.readthedocs.io/en/latest/building_applications
+from llama_stack_client import Agent
+from llama_stack_client.lib.agents.event_logger import EventLogger
+from llama_stack_client import LlamaStackClient
+from termcolor import cprint
+import argparse
+import logging
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+stream_handler = logging.StreamHandler()
+stream_handler.setLevel(logging.INFO)
+formatter = logging.Formatter('%(message)s')
+stream_handler.setFormatter(formatter)
+logger.addHandler(stream_handler)
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-r", "--remote", help="Uses the remote_url", action="store_true")
+parser.add_argument("-s", "--session-info-on-exit", help="Prints agent session info on exit", action="store_true")
+parser.add_argument("-a", "--auto", help="Automatically runs examples, and does not start a chat session", action="store_true")
+args = parser.parse_args()
+
+llama_model="meta-llama/Llama-3.2-3B-Instruct"
+granite_model="ibm-granite/granite-3.2-8b-instruct"
+# Connect to a llama stack server
+if args.remote:
+    base_url = os.getenv("REMOTE_BASE_URL")
+    mcp_url = os.getenv("REMOTE_MCP_URL")
+else:
+    base_url="http://localhost:8321"
+    mcp_url="http://host.containers.internal:8000/sse"
+
+client = LlamaStackClient(base_url=base_url)
+logger.info(f"Connected to Llama Stack server @ {base_url} \n")
+
+# Get tool info and register tools
+registered_tools = client.tools.list()
+registered_tools_identifiers = [t.identifier for t in registered_tools]
+registered_toolgroups = [t.toolgroup_id for t in registered_tools]
+
+if "mcp::slack" not in registered_toolgroups:
```

Comment:
If you are going to ensure the correct MCP servers are registered for this demo, do you want to also add the same if  statement for the openshift mcp server?